omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js

```diff
@@ -662,9 +663,14 @@ ConfirmationSectionModel.prototype.select = function() {
                     }
                     // question title is label, all fields on single line
                     else {
-                        summaryDiv.append("<p><span class='title'>" + question.title().text() + ": </span>"
+                        if (question.title().text() == "Name") {
+                            summaryDiv.append("<p><span class='title'>" + question.title().text() + ": </span>"
+                            + (question.valueAsTextNoSeparator) + "</p>");
+                        }
+                        else {
+                            summaryDiv.append("<p><span class='title'>" + question.title().text() + ": </span>"
                             + (question.valueAsText && !/^\s*$/.test(question.valueAsText) ? question.valueAsText : "--") + "</p>");
-
+                        }
```

Comment:
Are these formatting changes necessary?

Reply:
Can you please specify which specific formatting change you are referring to? We altered this code to specifically find the name value and remove the field separators in regards to RA-1352.